plugins/schema-discovery-default/src/main/java/com/thinkbiganalytics/discovery/parsers/hadoop/SparkFileSchemaParserService.java

```diff
@@ -92,12 +92,11 @@ public Schema doParse(InputStream inputStream, SparkFileType fileType, TableSche
             return toSchema(response.getResults(), fileType, tableSchemaType);
 
         } catch (Exception e) {
-            log.warn("Error parsing file {}", fileType);
+            log.error("Error parsing file {}: {}", fileType, e.getMessage());
             throw new IOException("Unexpected exception. Verify file is the proper format", e);
         } finally {
             tempFile.delete();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Do something with the "boolean" value returned by "delete". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS899)